src/sentry/tasks/integrations.py

```diff
@@ -154,3 +156,66 @@ def kick_off_status_syncs(project_id, group_id, **kwargs):
                 'external_issue_id': external_issue_id,
             }
         )
+
+
+@instrumented_task(
+    name='sentry.tasks.integrations.kickoff_vsts_subscription_check',
+    queue='integrations',
+    default_retry_delay=60 * 5,  # TODO(lb): not sure what this should be
+    max_retries=5,
+)
+@retry()
+def kickoff_vsts_subscription_check():
+    organization_integrations = OrganizationIntegration.objects.filter(
+        integration__provider='vsts',
+        # integration__status=ObjectStatus.VISIBLE,
+        # status=ObjectStatus.VISIBLE,
```

Comment:
I feel like status should matter, but I have a feeling that would be unreliable since deletion is not finished.